modules/renderer/features.js

```diff
@@ -506,6 +513,7 @@ export function rendererFeatures(context) {
 
 
     features.hasHiddenConnections = function(entity, resolver) {
+        if (!features.featureFitsDateRange(entity)) return true;
```

Comment:
This method is called to prevent, for example, moving a node that’s connected to a hidden feature. I’d think that whether the passed-in entity falls within the date range would be irrelevant to whether it’s connected to other hidden features. Granted, this edge case would only present itself in one edge case – the one where I tried to work around OpenHistoricalMap/issues#459 by entering a way ID into the search bar.

Reply:
> within the date range would be irrelevant to whether it’s connected to other hidden features

So long as we remember that "hidden" simply means "does not match a checkbox" and not "has been suppressed from the map."

I have removed this line per your instruction. This means that `hasHiddenConnections()` will instantly return false if all Map Features boxes are checked.